resources/js/processes/modeler/components/inspector/ScreenSelect.vue

```diff
@@ -1,151 +1,175 @@
-          .get("screens/" + value)
-          .then(response => {
-            this.loading = false;
-            this.content = response.data;
-          })
-          .catch(error => {
-            this.loading = false;
-            if (error.response.status == 404) {
-              this.content = '';
-              this.error = this.$t('Selected screen not found');
-            }
-          });
-      },
-      load(filter) {
-        let params = Object.assign(
-          {
-            type: this.type(),
-            interactive: this.interactive(),
-            order_direction: 'asc',
-            status: 'active',
-            selectList: true,
-            filter: (typeof filter === 'string' ? filter : '')
-          },
-          this.params
-        );
-        this.loading = true;
-        ProcessMaker.apiClient
-          .get('screens?exclude=config', {
-            params: params
-          })
-          .then(response => {
-            this.loading = false;
-            this.screens = response.data.data;
-          })
-          .catch(err => {
-            this.loading = false;
-          });
-      },
-      validate() {
-        if (!this.required || this.value && this.value !== undefined)  {
-          return;
-        }
-
-        this.error = this.$t('A screen selection is required');
-      },
+    interactive() {
+      if (this.params && this.params.interactive) {
+        return this.params.interactive;
+      }
+      return false;
+    },
+    loadScreen(value) {
+      this.loading = true;
+      ProcessMaker.apiClient
+        .get(`screens/${value}`)
+        .then(({ data }) => {
+          this.loading = false;
+          this.content = data;
+        })
+        .catch((error) => {
+          this.loading = false;
+          if (error.response.status === 404) {
+            this.content = "";
+            this.error = this.$t("Selected screen not found");
+          }
+        });
     },
-    mounted() {
-      this.validate();
-      if (this.value) {
-        this.loadScreen(this.value);
+    load(filter) {
+      const params = {
+        type: this.type(),
+        interactive: this.interactive(),
+        order_direction: "asc",
+        status: "active",
+        selectList: true,
+        filter: typeof filter === "string" ? filter : "",
+        ...this.params,
+      };
+      this.loading = true;
+      ProcessMaker.apiClient
+        .get("screens?exclude=config", {
+          params,
+        })
+        .then(({ data }) => {
+          this.loading = false;
+          this.screens = data.data;
+        })
+        .catch((err) => {
+          this.loading = false;
+        });
+    },
+    /**
+     * @param {Object} data - The response we get from the emitter
+     * @param {string} data.id - the screen id
+     * @param {string} data.assetType - The Asset type, ex: screen
+     */
+    processAssetCreation({ id, assetType }) {
+      if (assetType === "screen") this.$emit("input", id);
+    },
```

Comment:
process the emitter from the AssetQuickCreate component and sent an emit to the multiselect to select the value from the screen creaed